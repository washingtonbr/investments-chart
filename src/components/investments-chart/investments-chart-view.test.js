import React from 'react';
import renderer from 'react-test-renderer';
import InvestmentsChart from './investments-chart-view';
import {
  mockLabels,
  mockData,
} from './investments-chart-view.mock';

jest.mock('../../utils/date');

describe('InvestmentsChartView', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <InvestmentsChart
          labels={mockLabels}
          data={mockData}
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
