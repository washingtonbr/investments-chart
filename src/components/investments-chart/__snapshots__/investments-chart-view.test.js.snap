// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvestmentsChartView renders correctly 1`] = `
<canvas
  height={55}
  width={100}
/>
`;
